src/components/Error/index.js

```diff
@@ -0,0 +1,20 @@
+import { h, Component } from 'preact'
+import {errors} from '../strings/errors'
+import theme from '../Theme/style.css'
+import style from './style.css'
+
+const ApiUploadError = ({error}) => {
```

Comment:
Minor: I would call it `Error` instead of `ApiUploadError`, this way it's consistent with its folder. Also, since this component could be used for errors other than the api (considering there is nothing specific to the api inside of it), `Error` seems to make more sense.